Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -719,7 +724,7 @@ open class NavigationMapView: UIView {
         let shape = delegate?.navigationMapView(self, casingShapeFor: route) ?? defaultShape
         
         let geoJSONSource = self.geoJSONSource(shape)
-        let sourceIdentifier = route.identifier(.source(isMainRoute: isMainRoute, isSourceCasing: isMainRoute))
+        let sourceIdentifier = route.identifier(.source(isMainRoute: isMainRoute, isSourceCasing: true))
```

Comment:
For route casing layer source, the `isSourceCasing` should be `true`

Reply:
I wonder if it was this way because we used to only show traffic congestion on the main route but not any alternative routes? In any case, the change makes logical sense.